README.md

```diff
@@ -128,6 +128,19 @@ vars:
     quickbooks_union_databases: ['quickbooks_usa','quickbooks_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ``` 
 
+### Configuring Account Type Names
+QuickBooks does allow for the `account_type` and `account_subtype` field values to be configured for customers internally. While our current filtered logic within our intermediate models accounts for the default values, it's possible there are different or multiple values that might change what the account types would be. 
+
+If you have different field value filters you'd like to set, you will need to configure the `account_type` and `account_subtype` variables that account for these variables in your `dbt_project.yml`.
+
+```yml
+vars: 
+  quickbooks__accounts_payable_reference: accounts_payable_value # 'Accounts Payable' is the default filter set
+  quickbooks__accounts_receivable_reference: account_receivable_value # 'Accounts Receivable' is the default filter set
+  quickbooks__undeposited_funds_reference: account_undeposited_funds_value # 'UndepositedFunds' is the default filter set
+  quickbooks__sales_of_product_income_reference: account_sales_of_product_income_value # 'SalesOfProductIncome' is the default filter set
```

Comment:
I would specify with each if it is either on the `account_type` or `account_sub_type`

```suggestion
vars: 
  quickbooks__accounts_payable_reference: accounts_payable_value # 'Accounts Payable' is the default filter set for the account_type reference.
  quickbooks__accounts_receivable_reference: account_receivable_value # 'Accounts Receivable' is the default filter set for the account_type reference.
  quickbooks__undeposited_funds_reference: account_undeposited_funds_value # 'UndepositedFunds' is the default filter set for the account_sub_type reference.
  quickbooks__sales_of_product_income_reference: account_sales_of_product_income_value # 'SalesOfProductIncome' is the default filter set for the account_sub_type reference.
```

Reply:
Updated!